config/parameter.go

```diff
@@ -41,3 +47,38 @@ const (
 	ParameterTypeFile                              // file
 	ParameterTypeDuration                          // duration
 )
+
+func (pt ParameterType) MarshalText() ([]byte, error) {
+	return []byte(pt.String()), nil
+}
+
+func (pt *ParameterType) UnmarshalText(b []byte) error {
```

Comment:
In which situations are we going to unmarshal the parameter type?

Reply:
Good point, we don't actually need this, I took the code from the operation. I'll remove it!

Reply:
It's gone 👍